src/coreComponents/physicsSolvers/fluidFlow/proppantTransport/ProppantTransport.cpp

```diff
@@ -914,13 +914,13 @@ void ProppantTransport::updateCellBasedFlux( real64 const GEOSX_UNUSED_PARAM( ti
 
     FluxKernel::launchCellBasedFluxCalculation( stencilWrapper,
                                                 downVector,
-                                                flowAccessors.get< extrinsicMeshData::flow::pressure >(),
-                                                flowAccessors.get< extrinsicMeshData::flow::gravityCoefficient >(),
-                                                slurryFluidAccessors.get< extrinsicMeshData::slurryfluid::density >(),
-                                                slurryFluidAccessors.get< extrinsicMeshData::slurryfluid::viscosity >(),
-                                                permAccessors.get< extrinsicMeshData::permeability::permeability >(),
-                                                permAccessors.get< extrinsicMeshData::permeability::permeabilityMultiplier >(),
-                                                flowAccessors.get< extrinsicMeshData::elementAperture >(),
+                                                flowAccessors.pressure(),
+                                                flowAccessors.gravityCoefficient(),
+                                                slurryFluidAccessors.density(),
+                                                slurryFluidAccessors.viscosity(),
+                                                permAccessors.permeability(),
+                                                permAccessors.permeabilityMultiplier(),
+                                                flowAccessors.elementAperture(),
```

Comment:
This call (e.g.) gets definitely more readable.

Reply:
Yes, I agree that the kernel calls are lighter.